R/data.R

```diff
@@ -218,7 +218,8 @@
 
 #' Subject Visits Dataset
 #'
-#' A SDTM SV dataset from the CDISC pilot project
+#' A SDTM SV dataset from the CDISC pilot project. Duplicate observation for group
+#' variable USUBJID and VISIT is corrected.
```

Comment:
Please use backticks for variable names, i.e., `USUBJID` instead of USUBJID.

Reply:
Updated as suggested